integreat_cms/cms/templates/hix_widget.html

```diff
@@ -13,41 +14,51 @@
             <i icon-name="loader" class="animate-spin"></i> {% translate "Loading..." %}
         </div>
     </div>
-    <label>{% translate "Improve the understandability of the page content." %}</label>
-    <div id="hix-container" data-content-changed>
-        <canvas id="hix-chart"></canvas>
-    </div>
-    <div class="mb-2">
-        <div data-hix-state="updated" class="bg-green-400 rounded p-2 hidden">
-            <i icon-name="check-circle"></i>
-            {% translate "HIX value is up to date." %}
-        </div>
-        <div data-hix-state="outdated" class="bg-yellow-400 rounded p-2 hidden">
-            <i icon-name="alert-circle"></i>
-            {% translate "HIX value is outdated." %}
+    <label>{% translate "Improve the understandability of the page content" %}</label>
+    {% if request.user.expert_mode %}
+        {% render_field page_form.hix_ignore %}
+        <label for="{{ page_form.hix_ignore.id_for_label }}" class="secondary">{{ page_form.hix_ignore.label }}</label>
+        <div class="help-text">{{ page_form.hix_ignore.help_text }}</div>
+    {% elif page_form.hix_ignore.value %}
```

Comment:
```suggestion
    {% elif page_form.instance.hix_ignore %}
```
I think using the model instance here makes it more clear where the value is coming from